webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/50-reload-policy-file-server

```diff
@@ -11,4 +11,12 @@
 
 # Signal to cf-serverd that it shall look-up the new promises
 
+ACL_BACK="/var/rudder/tmp/cf-serverd.backup"
+ACL_FILE="/var/rudder/cfengine-community/inputs/common/1.0/cf-serverd.cf"
+if [ -f "${ACL_BACK}" ]; then
+  if [ "$(stat --format=%Y "${ACL_BACK}")" -ge "$(stat --format=%Y "${ACL_ACL_FILE}")" ]; then
+    exit 1
```

Comment:
don't we lack a "same date => exit 0" case?

Reply:
we should always exit 0 in fact